versioned/gc/core/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I could also see a situation where you put a core module that just exposes the AssetKeyConverter interface and then have the tiered core and non-tiered iceberg modules but it feels like the non-tiered core module will just be that one class, no?

Reply:
What about the classes in this module make them tiered to you? My motivation for this was it takes values and their referenced state and turns them into unreferenced assets. I expect most version store impls will require this as a separate step. The tiered part, to me, is finding the list of values and whether they are referenced

Reply:
I see what you're doing now. The abbreviated paths shown in github made it less than obvious. Agreed that the second phase of gc can be core as long as the first isn't. I think part of this that I find confusing is that this also makes spark a dependency of core gc. That was why I was making a comment about how few classes could actually be in core. I was thinking you wouldn't make core gc depend on Spark. It might even make sense to call this spark-gc or something (as opposed to core)

Reply:
Agreed, I think core is misleading. I have removed core and named the directory spark